packages/wp-now/src/config.ts

```diff
@@ -111,6 +114,7 @@ export default async function getWpNowConfig(
 		phpVersion: args.php as SupportedPHPVersion,
 		projectPath: args.path as string,
 		wordPressVersion: args.wp as string,
+		maxRequests: args.maxRequests as number,
```

Comment:
what is the purpose in choosing the max request count? is there a compelling reason to configure this or would it be just as good for us to pick a reasonable value and leave it at that?

what are the tradeoffs involved?

Reply:
Depending on how php is compiled/utilized, its possible for users to have issues at different levels of process-reuse. To account for that, I'd like to allow our users to tune this parameter to their own liking.

Reply:
One of the main tradeoffs is that during the switch from one instance to the next there seems to be some issue with keeping the cookies synced up correctly. The issue quickly resolves itself but will result in a couple of 403's when the handoff happens. The solution would be to find out why the instances aren't sharing sessions and correct that.

Reply:
> The issue quickly resolves itself but will result in a couple of 403's when the handoff happens

This seems like a blocking problem, a kind of major issue.

Reply:
@dmsnell Fixed, requires https://github.com/WordPress/wordpress-playground/pull/639

Reply:
I'm not a big fan of exposing that setting, but I also don't see any way to automatically use an optimal value as that will depend on many variables like hardware. Yup, let's just expose it.